common/tedge_utils/Cargo.toml

```diff
@@ -13,8 +13,7 @@ rumqttd = "0.7"
 rumqttlog = "0.7"
 tempfile = "3.2"
 thiserror = "1.0"
-tokio = { version = "1.9", default_features = false, features = ["signal", "sync", "macros"] }
-
+tokio = { version = "1.8", default_features = false, features = ["signal", "fs", "io-util", "macros"] }
```

Comment:
Plus `"rt-multi-thread"`.

Reply:
Also, version shouldn't be downgraded? The previous one is `1.9`, your change is `1.8`.